pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "pyuploadcare-example"
-version = "0.2.0"
+version = "4.2.0"
```

Comment:
Is it pyuploadcare or pyuploadcare-example version?

Reply:
pyuploadcare-example

Why don't we synchronize these versions?

Reply:
It's a good idea, I'm more than ok with it.